### ISSUE TITLE
Saved graph returns non-deterministic results

### ISSUE THREAD

**Issue by AndreyRub** [thtrieu/darkflow#27]
I've converted a YOLO/darknet graph using darkflow **--savepb** option.
I am loading it in another TensorFlow project and I manage to run it and get some data, which looks ok.
However, it seems that every time that I run it, I get slightly different results.
Maybe it is related to the dropout layer which has some randomness in it?  I've noticed that in "test" mode, darkflow is consistent, but could not find what I am missing in my own code that causes it to be deterministic in darkflow. 
Any advice?

**Comment by AndreyRub** [comment]
Indeed it was related to dropout, it was saved as 0.5 instead of 1.0

**Comment by thtrieu** [comment]
Can you elaborate on this? If this is a bug I am welcoming your pull requests!

**Comment by AndreyRub** [comment]
I am not sure if it is a bug or planned feature, but TensorFlow graphs saved using --savepb contain a dropout layer with a keep probability of 0.5.   For evaluation/testing purposes, it is not good, because

1) It makes the net "weaker"
2) It makes the results non deterministic, each run a bit different result.

It might be good if one is planning to re-train the network in another framework.

I don't know how to fix it correctly. Currently I hard-coded the keep probability to 1.0 when the layer is being built. That is of course not good for training. 
I guess that you can put a code right before the actual saving that changes the probability? 

> class dropout_layer(Layer):
>     def setup(self, p):
>         self.h['pdrop'] = dict({
>             #'feed': p, # for training
>             'feed': 1.0, # for training
>             'dfault': 1.0, # for testing
>             'shape': ()
>         })
>

**Comment by thtrieu** [comment]
Thanks for spotting this silly bug. Please pull the new code and check if it is working.
